src/server/game/Entities/Player/Player.h

```diff
@@ -1297,6 +1297,7 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         bool IsValidPos(uint8 bag, uint8 slot) const;
         bool HasBankBagSlot(uint8 slot) const;
         bool HasItemCount(uint32 item, uint32 count, bool inBankAlso = false) const;
+        bool HasItemCount(uint32 item) const { return HasItemCount(item, 1, false); };
```

Comment:
This function does not exists at TC?
Also you could just add a default arg to the existing one for the same effect

Reply:
Added default arg